packages/reports/tests/acceptance/navi-report-test.js

```diff
@@ -1653,6 +1653,116 @@ module('Acceptance | Navi Report', function(hooks) {
     assert.dom('.filter-values--current-period').hasText(`The current day. (${today})`, 'The current day');
   });
 
+  test('Date Picker start date beyond end date', async function(assert) {
+    assert.expect(19);
+
+    await visit('/reports/1');
+    await selectChoose('.filter-builder__select-trigger', 'Between');
+
+    assert.dom('.filter-values--date-range-input__low-value').hasText('Nov 09, 2015', 'The start date is Nov 09, 2015');
+    assert.dom('.filter-values--date-range-input__high-value').hasText('Nov 15, 2015', 'The end date is Nov 15, 2015');
+
+    //start date beyond end date
+    await clickTrigger('.filter-values--date-range-input__low-value');
+    await click($('.ember-power-calendar-day--current-month:contains(18)')[0]);
```

Comment:
Can you check if there's an attribute property with the day value on this element? We should try to avoid adding more jquery where possible.